### DISCUSSION TITLE
About 'enablement' of svg icons for commands used in view/title and editor/title command bars.

### DISCUSSION THREAD

**Discussion by boltex** [microsoft/vscode-discussions#1125] [Extension Development QnA] [answered]
the icons in  view/title and editor/title that were 'disabled' **when their enablement becomes false** used to be 'dimmed' (somewhat grayed out) automatically. (even when using SVG icons instead of product icons)

As an extension developer, _since the enablement clause now only changes the cursor appearance and not the icon_, I'm curious to know if I'm doing something wrong: **Does your extensions use such enablement clauses on some of the commands in your extensions?** And can you confirm that it does /or does not/ work in having a visible difference in icon appearance between enabled and disabled commands for SVG icons?

Thanks!

**Comment by boltex** [comment]
I would think that creating commands with custom icons and enablement clauses, and placing them in title bars of trees and editor would be common.... Curious to see if other extension developers have seen this problem. So I've opened a proper issue: https://github.com/microsoft/vscode/issues/209397

And made a sample repository to have an easy way to reproduce this bug: https://github.com/boltex/treeview-sample

**Comment by isidorn** [comment, marked as answer]
Sounds like a bug. Can you please file it here https://github.com/microsoft/vscode/issues
Thank you

**Reply by boltex** [reply]
Thanks, but that's exactly what I had written in the prior message above... ☝️  I've opened a proper issue: https://github.com/microsoft/vscode/issues/209397 , and  made a sample repository to have an easy way to reproduce this bug: https://github.com/boltex/treeview-sample  😄
